src/jit/assertionprop.cpp

```diff
@@ -2251,9 +2249,9 @@ GenTreePtr Compiler::optVNConstantPropOnTree(BasicBlock* block, GenTreePtr stmt,
 #ifdef _TARGET_64BIT_
             if (vnStore->IsVNHandle(vnCns))
             {
+#ifdef RELOC_SUPPORT
```

Comment:
Github shows this as moving the #ifdef, but really, the comment was moved into the ifdef.